src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/Buttons/index.tsx

```diff
@@ -5,70 +5,41 @@ import { useField, useFormState } from 'react-final-form'
 import Button from 'src/components/layout/Button'
 import Row from 'src/components/layout/Row'
 import { styles } from 'src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/style'
-import { createTxObject } from 'src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/utils'
+import { isReadMethod } from 'src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/utils'
 
 const useStyles = makeStyles(styles as any)
```

Comment:
is it possible to get rid of `any` there?

Reply:
Ok, this was fun [f049f85](https://github.com/gnosis/safe-react/pull/974/commits/f049f8598d5b6f3c5d9d9d3d97aad54f9ca3986d)

https://material-ui.com/styles/api/#createstyles-styles-styles